app/domain/authentication/authn_jwt/validate_generic_status_validations.rb

```diff
@@ -0,0 +1,56 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateGenericStatusValidations ||= CommandClass.new(
+      dependencies: {
+        validate_webservice_is_whitelisted: ::Authentication::Security::ValidateWebserviceIsWhitelisted.new,
+        validate_role_can_access_webservice: ::Authentication::Security::ValidateRoleCanAccessWebservice.new,
+        validate_webservice_exists: ::Authentication::Security::ValidateWebserviceExists.new,
+        enabled_authenticators: Authentication::InstalledAuthenticators.enabled_authenticators_str(ENV),
+        audit_log: ::Audit.logger
+      },
+      inputs: %i[authenticator_status_input]
+    ) do
+      extend(Forwardable)
+      def_delegators(:@authenticator_status_input, :authenticator_name, :account,
+                     :username, :webservice, :status_webservice)
+
+      def call
+        validate_user_has_access_to_status_webservice
```

Comment:
the status api is used to check the status of an authenticator, not the status of an authentication request. how is the role part of this check? do we check here an actual jwt token? if so - why?